example-utils/integration-test-utils/Cargo.toml

```diff
@@ -17,7 +17,6 @@ async-trait = { workspace = true }
 env_logger = { workspace = true }
 futures = { workspace = true }
 log = { workspace = true }
```

Comment:
doesn't look like integration-test-utils is a member of the workspace

Reply:
Done